chaos/manifests/network-partition-dm.yaml

```diff
@@ -0,0 +1,47 @@
+---
+# network partition between DM-worker and DM-master
+apiVersion: chaos-mesh.org/v1alpha1
+kind: NetworkChaos
+metadata:
+  name: network-partition-dm-worker-master
+  labels:
+    app: network-partition-dm-worker-master
+spec:
+  action: partition
+  mode: one
+  selector:
+    labelSelectors:
+      "app": "dm-worker"
+  direction: both
+  target:
+    selector:
+      labelSelectors:
+        "app": "dm-master"
+    mode: one
+  duration: "30s"
+  scheduler:
+    cron: "2-59/3 * * * *" # At every 3rd minute from 2 through 59, (2, 5, 8, ...)
+
+---
+# network partition between DM-master members
+apiVersion: chaos-mesh.org/v1alpha1
+kind: NetworkChaos
+metadata:
+  name: network-partition-dm-master-master
+  labels:
+    app: network-partition-dm-master-master
+spec:
+  action: partition
+  mode: one
+  selector:
+    labelSelectors:
+      "app": "dm-master"
+  direction: both
+  target:
```

Comment:
does this selector (`target` same as above `seletor`) match all dm-master, so each master is isolated from another?

Reply:
> does this selector (target same as above seletor) match all dm-master

YES. but we use `mode: one` here, so only one DM-master member will be isolated with another one (leader or follow, maybe we can change to `mode: all` to isolate the selected one with all other members?).

Reply:
Ah we have 3 master now. ~maybe we could use `fixed-percent` or `random-max-percent`~

I think either `one` or `all` is fine, they will test different scenario 😂  what's your opinion @GMHDBJD

Reply:
> I think either one or all is fine, they will test different scenario.
Agree. Maybe add more cases later.